orchestra/static/orchestra/timing/timecard/date-picker/date-picker.directive.es6.js

```diff
@@ -19,9 +19,16 @@ export default function datePicker (timeEntries) {
         minDate: scope.minDate ? scope.minDate.toDate() : null,
         maxDate: scope.maxDate ? scope.maxDate.toDate() : null,
         onSelect: function (date) {
-          scope.date = this.getMoment()
+          const newDate = this.getMoment()
```

Comment:
quick description of why we need this change?

Reply:
This keep the time constant when the date changes. In the old implementation, it would have reset the date to default (at 00:00).